COWorkflowsBasePlugin_1.4.iml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
From IDEAs side, consider to also add the patch number to the iml file:  COWorkflowsBasePlugin_1.4.1.iml
This way, IDEA can load multiple plugins more easy.